config/posts.json

```diff
@@ -2100,7 +2100,7 @@
   {
     "title": "AsyncAPI @Hacktoberfest",
     "date": "2020-09-30T05:00:00.000Z",
-    "type": "Marketing",
+    "type": "Community",
```

Comment:
Just realized we can probably add this file to `.gitignore` because it's generated on every build.